### PR TITLE
docs: add guide for jwt and acl plugin use

### PR DESCRIPTION
**Pull request by lostrouter**
**What this PR does / why we need it**:

Adds a guide for configuring the JWT and ACL plugin via kubernetes charts.   Demonstrates the use of kubernetes secrets to store RSA keys and ACL groups

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by artburkart** [comment]
This is awesome. We read through some of the code just the other day to reverse engineer the process of setting up an ACL, and it was on our TODO list to contribute docs for it, but you beat us to it with a PR that covers even more. 👍

**Comment by ryan4yin** [comment]
great!

**Comment by hbagdi** [comment]
1. Let's make sure the guide works as is. Users should be able to copy paste the commands and make it work. I don't see that being possible with the current guide.
2. Let's also add the steps to test the setup to prove that it works. See other guides on how it is done there.

**Comment by hbagdi** [comment]
@lostrouter A friendly ping to see if there is any progress here. This is really close and a very important doc that we would like to add.

**Comment by lostrouter** [comment]
@hbagdi i will set aside some time this week to to make the requested changes.

**Comment by lostrouter** [comment]
@hbagdi changes have been made.

File: docs/guides/configure-acl-plugin.md [hunk 1]
```diff
@@ -0,0 +1,694 @@
...
+  name: plain-user
+username: plain-user
+credentials:
+  - app-user-jwt
+" | kubectl apply -f -
+```
+
+## Use the credential
+
+Now to use a JWT to pass authentication. Let's store the user and admin jwt's in some environment variables. `USER_JWT` and `ADMIN_JWT`
```
Comment (hbagdi) [inline docs/guides/configure-acl-plugin.md:301]: Can you please add the steps of values that are set in these variables?

File: docs/guides/configure-acl-plugin.md [hunk 2]
```diff
@@ -0,0 +1,694 @@
...
+  - http:
+      paths:
+      - path: /post
+        backend:
+          serviceName: httpbin
+          servicePort: 80
+' | kubectl apply -f -
+```
+
+Any requests matching the proxying rules for `demo-get` and `demo` post will now require a valid JWT and the consumer for the JWT to be associate with the right ACL.
```
Comment (hbagdi) [inline docs/guides/configure-acl-plugin.md:191]: Can you update the entire doc to respect the 80 char column limit? Please wrap around long sentences.

Comment (lostrouter) [reply docs/guides/configure-acl-plugin.md:191]: done

File: docs/guides/configure-acl-plugin.md (hunk 1 above)
Comment (lostrouter) [reply docs/guides/configure-acl-plugin.md]: done

**Comment by hbagdi** [comment]
@lostrouter  Please fill in the contributor form to claim your Kong swag: https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#contributor-t-shirt

**Pull request merged**